### PR TITLE
Add Product list Page

### PR DESCRIPTION
**Pull request by rahulcs754**
- Added  Product Page
image
image

Please check and give suggestion

### CODE REVIEW

**Review by kjais1720 — COMMENTED** [review]
Already reviewed in the PR #4 for Signup page

**Review by kjais1720 — APPROVED** [review]

File: pages/product_list.html [hunk 1]
```diff
@@ -0,0 +1,294 @@
...
+                    <div class="col-sm-4">
+                        <div class="card card-overlay">
+                            <img src="../assets/img/list_img/7.jpg" class="card-image img-c" alt="Card-Image">
+                            <i class="card-close-icon far fa-times-circle"></i>
+                            <div class="card-header">
+                            <div class="card-title">Soho Watch</div>
+                            </div>
+                            <div class="card-info">
+                            <p class="card-price">Rs.1800</p>
+                            <p class="mrp-price">Rs.3000</p>
```
Comment (ttahm3d) [inline pages/product_list.html]: Use HTMl code for Rupee symbol instead of using Rs. `&#8377;`

File: pages/product_list.html [hunk 2]
```diff
@@ -0,0 +1,294 @@
...
+                        <div class="card card-overlay">
+                            <img src="../assets/img/list_img/29.jpg" class="card-image img-c" alt="Card-Image">
+                            <i class="card-close-icon far fa-times-circle"></i>
+                            <div class="card-header">
+                            <div class="card-title">Soho Bry Watch</div>
+                            </div>
+                            <div class="card-info">
+                            <p class="card-price">Rs.1800</p>
+                            <p class="mrp-price">Rs.3000</p>
+                            <p class="card-discount">60% off</p>
```
Comment (ttahm3d) [inline pages/product_list.html:240]: Discount value is not correct. Not that it matters but change it if you can.

**Review by ttahm3d — APPROVED** [review]
Added few comments. Which can be changed later.

File: pages/product_list.html (hunk 1 above)
Comment (rahulcs754) [reply pages/product_list.html]: i changed my rupees to doller

File: pages/product_list.html (hunk 2 above)
Comment (rahulcs754) [reply pages/product_list.html:240]: it is correct 
3000 * 60 /100

**Pull request merged**